ci-operator/jobs/jboss-fuse/fuse-xpaas-qe-container/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- jboss-fuse-approvers
+reviewers:
+- jboss-fuse-approvers
```

Comment:
Please add `cspi-qe-ocp-lp` as both approvers and reviewers in jobs folder. As config file updates jobs file also, so we should be able to approve this also.